identity-service/src/models/userevents.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+module.exports = (sequelize, DataTypes) => {
+  const UserEvents = sequelize.define('UserEvents', {
+    walletAddress: {
+      type: DataTypes.STRING,
+      primaryKey: true,
+      references: { model: 'Users', key: 'walletAddress' }
+    },
+    needsRecoveryEmail: {
+      allowNull: true,
+      type: DataTypes.BOOLEAN
```

Comment:
@raymondjacobson you can set `defaultValue` to true here https://sequelize.org/master/manual/models-definition.html

Reply:
I didn't set defaultValue to true here because the default should probably not be true. The table as a whole could store other events that need some sort of client action (let's say authMigration). For a user that needs an auth migration, they don't necessarily need a recovery email.

The reason I made this field a "truthy" one (instead of recoveryEmailSent and setting it to true on the route) was because we can just stop writing to this field and make no client changes. In the case of recoveryEmailSent, if we decided to stop writing to the field (or drop the column), we would have to make client changes. this way, we don't

Reply:
@raymondjacobson fair point. i was thinking these colums would be idempotent but if you think it would be better to lay it out this way that's fine